spec/models/interest_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Interest do
+  it 'has none to begin with' do
+    expect(Interest.count).to eq 0
+  end
+
+  it 'can by created with valid arguments' do
+    user = create(:user)
+    Interest.new(name: 'jokes', type: 'Hobby', user_id: user.id).save
+
+    expect(Interest.count).to eq 1
```

Comment:
Assuming the number of records in the DB is not a very good practice neither. I believe you can make interest factory work without additional parameters and use a spec with the FactoryBot lint instead of that one.
Something along lint_spec.rb
```ruby
RSpec.describe 'FactoryBot linter' do
  it 'verifies that all FactoryBot factories are valid' do
    FactoryBot.lint
  end
end
```
It will try to create a record for each factory with the default parameters, and it will fail if any of the cannot be created - hence making the spec above obsolete 😄